web/packages/teleport/src/Integrations/Enroll/MachineIDIntegrationSection.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AnsibleIcon,
+  CircleCIIcon,
+  GitHubIcon,
+  GitLabIcon,
+  JenkinsIcon,
+  ServersIcon,
+} from 'design/SVGIcon';
+import { Box, Flex, Link as ExternalLink, Text } from 'design';
+import React from 'react';
+
+import {
+  IntegrationEnrollEvent,
+  IntegrationEnrollKind,
+  userEventService,
+} from 'teleport/services/userEvent';
+import { IntegrationTile } from 'teleport/Integrations/Enroll/common';
```

Comment:
We import from the same directory, so it is better to use relative import.

```suggestion
import { IntegrationTile } from './common';
```